src/main/java/org/apache/commons/lang3/reflect/MethodUtils.java

```diff
@@ -126,13 +149,41 @@ public static Object invokeMethod(final Object object, final String methodName,
     /**
      * <p>Invokes a named method whose parameter type matches the object type.</p>
      *
-     * <p>This method delegates the method search to {@link #getMatchingAccessibleMethod(Class, String, Class[])}.</p>
+     * <p>This method supports calls to methods taking primitive parameters 
+     * via passing in wrapping classes. So, for example, a {@code Boolean} object
+     * would match a {@code boolean} primitive.</p>
+     *
+     * <p>This is a convenient wrapper for
+     * {@link #invokeMethod(Object object,String methodName, Object[] args, Class[] parameterTypes)}.
```

Comment:
I guess this should be ``invokeMethod(final Object object, final boolean forceAccess, final String methodName, Object[] args, Class[] parameterTypes)`?